build/images/scripts/install_cni

```diff
@@ -2,25 +2,46 @@
 
 set -euo pipefail
 
+# Fetching the list of the binaries that user wants to skip installing.
+arguments=$SKIP_CNI_BINARIES
+oldDelimeter=$IFS
+IFS=','
+flags=$arguments
+binaries=()
+for args in $flags; do
+  binaries+=($args)
+done
+IFS=$oldDelimeter
```

Comment:
The above can be written in one line:
```
IFS=',' read -r -a binaries <<< "$SKIP_CNI_BINARIES"
```

Reply:
I think by specifying IFS before the command, it only affects that line. Maybe you can test to verify it.